lib/printer.js

```diff
@@ -1101,11 +1105,19 @@ function genericPrintNoParens(path, options, print) {
         return concat(parts);
 
     case "ClassProperty":
-        return concat([
-            path.call(print, "key"),
-            path.call(print, "typeAnnotation"),
-            ";"
-        ]);
+        var parts = [];
+        if (n.static)
+            parts.push("static ");
```

Comment:
nice! I'd reserve adding braces for a separate pull request though, just following your conventions. How do you feel about spaces instead of tabs? :P